### PR TITLE
Daily coding practice 13 is added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/objectOrientedProgram/Book.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+public class Book {
+    private String title;
+    private String author;
+    private String ISBN;
+    private static ArrayList<Book> bookCollection = new ArrayList<Book>();
+
+    public Book(String title, String author, String ISBN) {
+        this.title = title;
+        this.author = author;
+        this.ISBN = ISBN;
```
Comment (Ams12) [inline simple/src/objectOrientedProgram/Book.java:14]: Give proper variable name.

File: simple/src/objectOrientedProgram/Store_ReadMe [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ and methods to add and remove books from a collection.
+Output:
+ List of book is :
+ Book Name : C language  By  :deepak  ISBN : 123456
+ Book Name : Core java By  :santosh  ISBN : 132165
+
+ After removing the book the book list is
+ Book Name : Core java By  :santosh  ISBN : 132165
+
+ Process finished with exit code 0
```
Comment (Ams12) [inline simple/src/objectOrientedProgram/Store_ReadMe:11]: Delete this line and remove the extra space.

File: simple/src/objectOrientedProgram/Store.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        for (Book book : bookCollection) {
+            System.out.println("Book Name : " + book.getTitle() + " By  :" + book.getAuthor() + "  ISBN : " + book.getISBN());
+        }
+        System.out.println(" ");
+        Book.remove_Book(book1);
+        System.out.println("After removing the book the book list is ");
+        for (Book book : bookCollection) {
+            System.out.println("Book Name : " + book.getTitle() + " By  :" + book.getAuthor() + "  ISBN : " + book.getISBN());
+        }
+
```
Comment (Ams12) [inline simple/src/objectOrientedProgram/Store.java:25]: Remove space.

**Review by Ams12 — CHANGES_REQUESTED** [review]

File: simple/src/objectOrientedProgram/Store_ReadMe (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/objectOrientedProgram/Store_ReadMe]: done

File: simple/src/objectOrientedProgram/Book.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/objectOrientedProgram/Book.java]: done

**Review by Ams12 — APPROVED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**